Fix elasticsearch-secret template port default function

Sonins opened the pull request:
<!--
Thank you for contributing! Please make sure that your code changes
are covered with tests. And in case of new features or big changes
remember to adjust the documentation.

Feel free to ping committers for the review!

In case of existing issue, reference it using one of the following:

closes: #ISSUE
related: #ISSUE

How to write a good git commit message:
http://chris.beams.io/posts/git-commit/
-->

Default function in helm template should be "default DEFAULT_VALUE GIVEN_VALUE".
Original code can invoke elasticsearch host configuration error.

Refer `Using the default function` section below link.
https://helm.sh/docs/chart_template_guide/functions_and_pipelines/


---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/main/UPDATING.md).

Sonins committed:
Fix elasticsearch-secret template port default function

jedcunningham reviewed (changes requested):
Good catch @Sonins! Can you add test coverage for this? This is an example you could follow:
https://github.com/apache/airflow/blob/main/chart/tests/test_metadata_connection_secret.py

Sonins committed:
Add elasticsearch secret non-default port test function.

jedcunningham commented on chart/tests/test_elasticsearch_secret.py:
```diff
@@ -96,3 +96,24 @@ def test_should_correctly_handle_password_with_special_characters(self):
+        assert (
+            "http://username%21%40%23$%25%25%5E&%2A%28%29:password%21%40%23$%25%25%5E&%2A%28%29@"
+            "elastichostname:9200" == connection
+        )
```
```suggestion
                        "host": "elastichostname",
                        "port": 2222,
                    },
                }
            }
        )

        assert "http://elastichostname:2222" == connection
```

Let's keep it simple and also use a non-default port?

Sonins replied:
Good. I committed it.

Sonins replied:
OK, It turns out there is a little problem.
Without user and pass field, `<no+value>` is included in url string, like this.
`http://<no+value>:<no+value>@elastichostname:2222`

It makes test fail.

![image](https://user-images.githubusercontent.com/26663842/128814721-114465cc-92aa-4b02-9c58-195c33d3aeed.png)

My mistake, I should've tested for this before i committed it, sorry.

For now I can think 2 options.

1. Insert simple username and pass into test function.
2. Fix something that makes user and pass in url string <no+value>. And make default username something like 'root', instead.

jedcunningham replied:
Ugh, no worries, just put in simple values for now. Might be another follow up we need to look at.

Sonins replied:
Resolve: https://github.com/apache/airflow/pull/17428/commits/ef2e80b6fe6c747b040ea7f4954f8ae95a7aefa1

Sonins committed:
Update chart/tests/test_elasticsearch_secret.py

Co-authored-by: Jed Cunningham <66968678+jedcunningham@users.noreply.github.com>

Sonins committed:
Add simple username and password to test coverage.

Sonins committed:
Fix formatting

Previous check failed static-check because it is not correctly formatted.
Fixing it.
